### PR TITLE
support regression models

### LINKED ISSUE #615: Support regression model
**Is your feature request related to a problem? Please describe.**

SQLFlow needs to support the regression model, the label type can be INT/FP32/FP16 which derived by the column type, and we also need an e2e test for the regression model.

### PR DESCRIPTION
**Pull request by typhoonzero**
Support regression models.

NOTE, reading a table with all float columns using `mysql-connector` will cause infinite reading so switched to use `mysqlclient`, same issue described here: https://bugs.mysql.com/bug.php?id=91971

fixed #615

### CODE REVIEW

File: sql/python/sqlflow_submitter/db.py [hunk 1]
```diff
@@ -53,11 +54,15 @@ def reader():
...
+        while True:
+            rows = cursor.fetchmany(size = fetch_size)
+            if not rows:
+                break
             # NOTE: keep the connection while training or connection will lost if no activities appear.
-            if driver == "mysql" and not conn.is_connected():
-                conn.ping(True)
+            # if driver == "mysql" and not conn.is_connected():
+            #     conn.ping(True)
+            print("fetched rows: ", len(rows))
```
Comment (Yancey0623) [inline sql/python/sqlflow_submitter/db.py]: Please remove the debug code.

File: sql/codegen.go [hunk 2]
```diff
@@ -161,10 +161,32 @@ func newFiller(pr *extendedSelect, ds *trainAndValDataset, fts fieldTypes, db *D
 		}
 	}
 
-	// FIXME(typhoonzero): support soft label in addition to int types
+	// for k, v := range fts {
```
Comment (Yancey0623) [inline sql/codegen.go]: Please delete the unused code.

File: sql/python/sqlflow_submitter/db.py [hunk 3]
```diff
@@ -16,13 +16,14 @@
 
 def connect(driver, database, user, password, host, port):
     if driver == "mysql":
-        from mysql.connector import connect
+        # NOTE: use MySQLdb to avoid bugs like infinite reading:
```
Comment (Yancey0623) [inline sql/python/sqlflow_submitter/db.py:19]: Add a unit test for the infinite bug or merge https://github.com/sql-machine-learning/sqlflow/pull/611 firstly?

Comment (typhoonzero) [reply sql/python/sqlflow_submitter/db.py:19]: Done.

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**